src/Helper.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Helper.hpp"
+
+const std::string Helper::INPUT_FILES_DIRECTORY = "test_files";
+
+bool Helper::CreateTestFiles()
+{
+    //TODO: add Error handling
+    fs::create_directories(INPUT_FILES_DIRECTORY);
+    fs::path p1;
```

Comment:
p1 is an example of a bad variable name. It doesn't mean anything. The same in test cases. BTW. Why not initialize it immediately instead of one line below?